extensions-core/azure-extensions/src/test/java/org/apache/druid/storage/azure/AzureDataSegmentKillerTest.java

```diff
@@ -70,7 +73,7 @@ public class AzureDataSegmentKillerTest extends EasyMockSupport
       ImmutableMap.of("containerName", CONTAINER_NAME, "blobPath", BLOB_PATH),
       null,
       null,
-      NoneShardSpec.instance(),
+      new LinearShardSpec(0),
```

Comment:
`NoneShardSpec` is marked deprecated, and it doesn't really matter for these tests.